bitcoinology/what-is-lightning-network.md

```diff
@@ -0,0 +1,90 @@
was showing them around and they said they gave him, I think they gave him, I know, they gave him Moon Wallet, he installed it. And I just met them on the street and I said, oh, you're selling coffee, I'll buy coffee with Bitcoin from you. So, it's a meme, right, You have to buy coffee with Bitcoin. You've got it, right? So he goes into the building, comes back out with a bag of coffee, which you can't quite see, but I'm holding it in my hand here. And he says, $10. And I paid him and it worked immediately. Now, I think that's a really cool example of how it can work. This is how it could be. It could be the case we could reach a point where everyone can reliably pay each other. $10.01 I paid for a $10 bag of coffee. I didn't know him, he didn't know me. It worked perfectly in literally like two or three seconds. Of course, it's not always going to be great like that, But that was an example where it was really good. Another example, it is a few seconds longer, but also a short video, is this, I actually just nabbed this from Twitter. This is, a lot of people here will know him. We're getting a bit of order at the company. We're a couple of minutes late. So it's what you'd expect, it's the normal McDonald's... That's a bit... Squeeze... That one's squeezed... Okay, that's water. Right, paint. And then that seems to shut down. And here... That's a bit of a break-up, we know that's what's going on. And with Bitcoin, see that? I'm going to get the seats jacked out. I'm going to put it here. Because I'm going to break up. We know that's not true. So with Bitcoin, see that? Yeah, we're going to see that. Yeah, nice one, I'm there. And I'm going to use my Allen bits wallet. Guys, like, thanks for the tips, but this is not the Allen bits tip jar. I think this is a good excuse to make a payment. I'm not a tip jar, I mean it's not bad to make a payment from a tip jar. There we are. Third big coin. I wonder what would be a coffee from McDonald's using like... There you go. Seven points. Scare it. Don't pay it. Just please. I want you to copy a copy from McDonald's using lightning. I know. All right, here we go. Is that 10 points? Ha, ha, ha, ha, ha, ha, ha, ha. Scan it. Double-click it. It's really slow. Come on, lightning. You can do it. LN bits will let. No, I mean it's just a little bit slow, but it works fine. Oh wow, look. Yeah, it does. Really? It works just fine, yeah. The paper works through. Yeah. Wow. Nice. Nice. Thanks for the company. And thanks for letting me pay for it. I'll get the LN bits, Chuck. So, I mean, that was a bit slower, but in my experience, I used the same thing several times and it was absolutely fine. Okay, well, I think I'm more or less done in the first half, so we can, if anybody has any questions or thoughts, please Ask, otherwise, oops, there we go. Otherwise we can move on to actually using it. Yeah, your turn. So this is your turn. So, yeah, questions. Between the nodes you can add liquidity, what did you mean by that? 
```

Comment:
```suggestion
I'm not going to mention some of these other things. This is the Starbucks point of sale, which in my experience worked flawlessly every time. I think it's Ibex, does anyone know? I don't really care, it's some company that did it. It's kind of funny though, they show three different denominations. They throw dollars and then they show SATs, S-A-T, capital, and then they show BTC as well, just in case you get confused. It's just supposed to help you understand what the price is, I guess. But anyway, it works absolutely fine with more than one Lightning Wallet for me. Also McDonald's, which I'll show an example in a minute. Just to make pictures, like that's Bitcoin, this is my slideshow now, my holiday, I mean, it's a bit, sorry. Bitcoin Beach, it looks really nice in the sunset, cool. This is Hope House in El Zonte where you have the, some guys drove from Argentina or something, I don't even know the story here. This is something I just donated to, And that was the new mural by Bitrefill. It's a very interesting place to have a look. But San Salvador was different because they didn't have that Bitcoin beach wallet thing. So a lot of people were using Chivo and didn't really work very well with merchants. But On the other hand, McDonald's and Starbucks and a few chain restaurants were working fine. So it was all very strange. And you talk to the taxi drivers, half of them think it's great, half of them think it's terrible. I don't know what that proves. Yeah, so video time. So I just thought it might be amusing to finish this off by showing you what it's like to use it. So, if my eyesight can actually find the... There it is. So, this was a completely serendipitous meeting on the street. It's already scanned the QR code. All we need is a hotspot and 815. You got it? Woo! All right, all right, there we go. Every time I saw it, everybody's like, woo! Strict deals, coffee. I keep on talking about coffee, tea. So this very short video is basically, I met a guy called Tomer on the street. He's kind of a local here, actually. He's a journalist and videos and stuff. Him and his cameraman guy were talking to this guy who runs a coffee plantation, or owns a coffee plantation, I don't know. Anyway, he was getting interested in Bitcoin because he was showing them around and they said they gave him, I think they gave him, I know, they gave him Moon Wallet, he installed it. And I just met them on the street and I said, oh, you're selling coffee, I'll buy coffee with Bitcoin from you. So, it's a meme, right, You have to buy coffee with Bitcoin. You've got it, right? So he goes into the building, comes back out with a bag of coffee, which you can't quite see, but I'm holding it in my hand here. And he says, $10. And I paid him and it worked immediately. Now, I think that's a really cool example of how it can work. This is how it could be. It could be the case we could reach a point where everyone can reliably pay each other. $10.01 I paid for a $10 bag of coffee. I didn't know him, he didn't know me. It worked perfectly in literally like two or three seconds. Of course, it's not always going to be great like that, But that was an example where it was really good. Another example, it is a few seconds longer, but also a short video, is this, I actually just nabbed this from Twitter. This is, a lot of people here will know him. We're getting a bit of order at the company. We're a couple of minutes late. So it's what you'd expect, it's the normal McDonald's... That's a bit... Squeeze... That one's squeezed... Okay, that's water. Right, paint. And then that seems to shut down. And here... That's a bit of a break-up, we know that's what's going on. And with Bitcoin, see that? I'm going to get the seats jacked out. I'm going to put it here. Because I'm going to break up. We know that's not true. So with Bitcoin, see that? Yeah, we're going to see that. Yeah, nice one, I'm there. And I'm going to use my Allen bits wallet. Guys, like, thanks for the tips, but this is not the Allen bits tip jar. I think this is a good excuse to make a payment. I'm not a tip jar, I mean it's not bad to make a payment from a tip jar. There we are. Third big coin. I wonder what would be a coffee from McDonald's using like... There you go. Seven points. Scare it. Don't pay it. Just please. I want you to copy a copy from McDonald's using lightning. I know. All right, here we go. Is that 10 points? Ha ha. Scan it. Double-click it. It's really slow. Come on, lightning. You can do it. LN bits will let. No, I mean it's just a little bit slow, but it works fine. Oh wow, look. Yeah, it does. Really? It works just fine, yeah. The paper works through. Yeah. Wow. Nice. Nice. Thanks for the company. And thanks for letting me pay for it. I'll get the LN bits, Chuck. So, I mean, that was a bit slower, but in my experience, I used the same thing several times and it was absolutely fine. Okay, well, I think I'm more or less done in the first half, so we can, if anybody has any questions or thoughts, please Ask, otherwise, oops, there we go. Otherwise we can move on to actually using it. Yeah, your turn. So this is your turn. So, yeah, questions. Between the nodes you can add liquidity, what did you mean by that? 
```